### ISSUE TITLE
Visually distinguish between Panels and Covers attached to frames

### ISSUE THREAD

**Issue by busti** [MrTJP/ProjectRed#1600]
It is impossible to tell wether a panel or a cover is attached to a frame since they look the same when made from the same material. It would be nice to have a visual indication between the two, i.e. extra visible wood pixels when a panel is attached.

**Comment by MrTJP** [comment]
Agreed. There should be an extra visual element on the face so it's more obvious.
